manifests/4.2/cluster-logging.v4.2.0.clusterserviceversion.yaml

```diff
@@ -61,8 +61,6 @@ metadata:
 spec:
   displayName: Cluster Logging
 
-  replaces: clusterlogging.v4.1.0
```

Comment:
@jwforres can you confirm we don't need this anymore?

Reply:
@richm @ewolinetz unless you guys still need/want it for your manual/CI upgrade testing?

Reply:
i feel like `replaces` is just causing issues currently and it seems that we can just use `skipRange` instead so i'm good with this being removed